### ISSUE TITLE
Two arguments audio_bitrate and video_bitrate are not working as expected.

### ISSUE THREAD

**Issue by Drachenfels** [kkroening/ffmpeg-python#244]
My code is simple. Load test.wav, drop channel, preserve all other stuff, save to file. However as it happens number of frames goes total bollocks I did some debugging and it seems python-ffmpeg passes wrong information to ffmpeg.

First my script:

```
import ffmpeg
import io
import wave

with open('test.wav', 'rb') as fpl:
    input_file = io.BytesIO()

    input_file.write(fpl.read())

    input_file.seek(0)

output_file = io.BytesIO()

with wave.open(input_file) as fpl:
    frames = fpl.getnframes()
    rate = fpl.getframerate()
    duration = '00:00:{}'.format(frames / float(rate))

input_file.seek(0)

print('DURATION:', duration)
print('FRAMES:', frames)
print('RATE:', rate)

# ffmpeg -i test.wav -f wav -ac 1 -acodec pcm_s16le -ar 8000 -ab 256k -b 256k
# --aframes 35840 -loglevel 56 yolo2.wav

process = (
    ffmpeg
    .input('pipe:')
    .output(
        'pipe:', format='wav', ac=1, acodec='pcm_s16le',
        ar=rate,
        audio_bitrate='128000',
        video_bitrate='128000',
        aframes=frames,
        loglevel=56,
    )
    .run_async(pipe_stdin=True, pipe_stdout=True)
)

output, err = process.communicate(input=input_file.read())

output_file.write(output)

output_file.seek(0)

with open('yolo.wav', 'wb') as fpl:
    fpl.write(output_file.read())
```

It should do exactly same stuff as:

ffmpeg -i test.wav -f wav -ac 1 -acodec pcm_s16le -ar 8000 -ab 256k -b 256k -aframes 35840 -loglevel 56 yolo2.wav

Now output from ffmpeg when using python-ffmpeg:

```
Splitting the commandline.
Reading option '-i' ... matched as input url with argument 'pipe:'.
Reading option '-f' ... matched as option 'f' (force format) with argument 'wav'.
Reading option '-b:v' ... matched as option 'b' (video bitrate (please use -b:v)) with argument '128000'.
Reading option '-b:a' ... matched as option 'b' (video bitrate (please use -b:v)) with argument '128000'.
Reading option '-ac' ... matched as option 'ac' (set number of audio channels) with argument '1'.
Reading option '-acodec' ... matched as option 'acodec' (force audio codec ('copy' to copy stream)) with argument 'pcm_s16le'.
Reading option '-aframes' ... matched as option 'aframes' (set the number of audio frames to output) with argument '35840'.
Reading option '-ar' ... matched as option 'ar' (set audio sampling rate (in Hz)) with argument '8000'.
Reading option '-loglevel' ... matched as option 'loglevel' (set logging level) with argument '56'.
Reading option '-t' ... matched as option 't' (record or transcode "duration" seconds of audio/video) with argument '00:00:4.48'.
Reading option 'pipe:' ... matched as output url.
```

Basically audio_format is traslated to -b while video_format is translated to -b.

**Comment by Drachenfels** [comment]
To focus on the issue.

When using python-ffmpeg:

```
Reading option '-b:v' ... matched as option 'b' (video bitrate (please use -b:v)) with argument '128000'.
Reading option '-b:a' ... matched as option 'b' (video bitrate (please use -b:v)) with argument '128000'.
```

When using ffmpeg directly:

```
Reading option '-ab' ... matched as option 'ab' (audio bitrate (please use -b:a)) with argument '256k'.
Reading option '-b' ... matched as option 'b' (video bitrate (please use -b:v)) with argument '256k'.
```

**Comment by Drachenfels** [comment]
Ha lol nvm, it's actually bug on ffmpeg itself.
